src/app/stores/migrator_baseline.ts

```diff
@@ -0,0 +1,278 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Nitpick: This seems like a good candidate to unit test

Reply:
Yes, it is being unit tested 🤓 The https://github.com/microsoft/charticulator/blob/master/src/tests/unit/charts/side-by-side.json chart has guides that get migrated. 🐱‍🐉